deblur/test/test_workflow.py

```diff
@@ -186,6 +188,72 @@ def test_dereplicate_seqs(self):
 
         self.assertEqual(act, exp)
 
+    def test_filter_minreads_samples_from_table(self):
+        """ Test filter_minreads_samples_from_table() function
+        for removal of samples with small number of reads
+        using the s4 dataset biom table
+        """
+        input_biom_file = join(self.test_data_dir, 'final.s4.biom')
+        table = load_table(input_biom_file)
+
+        # test basic filtering with 0 reads does not remove ok sample
+        new_table = filter_minreads_samples_from_table(table)
+        self.assertEqual(new_table.shape[1], 1)
```

Comment:
since we have 1 sample in the test table, enough to see if it is filtered away or not...